helm/kibana-secret/makefile

```diff
@@ -0,0 +1,17 @@
+default: test
+include examples.mk
+RELEASE := helm-kibana-security
+
+install:
+	helm upgrade --wait --timeout=$(TIMEOUT) --install --values values.yaml $(RELEASE) ../../
+
+test: secrets install goss
+
+purge:
+	kubectl delete secret kibana || true
+	helm del $(RELEASE)
+
+secrets:
+	encryptionkey=$$(docker run --rm busybox:1.31.1 /bin/sh -c "< /dev/urandom tr -dc _A-Za-z0-9 | head -c50") && \
+  kubectl create namespace paymenthub || echo "namespace already exists"  && \
```

Comment:
can we externalize these namespace variables or make it as a global variable and set it only once?

Reply:
This line works for creating secrets in specific NS. Just need to change the <namespace> in makefile before running the pipeline.